scripts/exglobal_stage_ic.py

```diff
@@ -30,7 +30,7 @@ def main():
     for key in keys:
         # Make sure OCNRES is three digits
         if key == "OCNRES":
-            stage.task_config.OCNRES = f"{stage.task_config.OCNRES :03d}"
+            stage.task_config.OCNRES = f"{stage.task_config.OCNRES:03d}"  # noqa: E203
```

Comment:
I don't think the `noqa: E203` is needed here since the `:` issue has been fixed.

```suggestion
            stage.task_config.OCNRES = f"{stage.task_config.OCNRES:03d}"
```